data/event-schemas/schema/m.space.child.yaml

```diff
@@ -9,8 +9,6 @@ properties:
         type: array
         description: |-
           A list of servers to try and join through. See also: [Routing](/appendices/#routing).
-
-          When not present or invalid, the child room is not considered to be part of the space.
```

Comment:
While the fields are required, the text being removed here is critical for operation of the feature. This is why when the spec PR was originally written it did not flag the fields as required.